rubygems-update.gemspec

```diff
@@ -23,6 +23,7 @@ Gem::Specification.new do |s|
   s.licenses = ["Ruby", "MIT"]
 
   s.files = File.read(File.expand_path("Manifest.txt", __dir__)).split
+  s.files += %w[sbom/rubygems.cdx.json]
```

Comment:
Already listed in `Manifest.txt` so not needed? I wonder if cycloneDX specification has some replacement for our Manifest.txt file, i.e., some section to list all the files in the package. Perhaps https://cyclonedx.org/docs/1.6/json/#components_items_data, but not sure.